app/clients/zendesk_sell_client.py

```diff
@@ -0,0 +1,94 @@
+import json
+import requests
+
+from typing import Dict, List, Union
+
+from flask import current_app
+
+from app.authentication.bearer_auth import BearerAuth
+from app.user.contact_request import ContactRequest
+
+__all__ = [
+    'ZenDeskSellClient'
+]
+
+
+class ZenDeskSellClient(object):
+
+    def _generate_lead_data(self) -> Dict[str, Union[str, List[str], Dict]]:
+
+        # The API and field definitions are defined here: https://developers.getbase.com/docs/rest/reference/leads
+
+        # validation based upon api mandatory fields
+        assert len(self.contact.name) or len(self.contact.department_org_name), 'Name or Org are mandatory field'
+
+        recipients = {
+            'internal': 'Colleagues within your department (internal)',
+            'external': 'Partners from other organizations (external)',
+            'public': 'Public'
+        }
```

Comment:
Do we need to translate to sentences in Zendesk Sell or are they okay to have keys for recipients?

Reply:
We need to have the exact naming as what's in zendesk sell, otherwise the tick box within the UI will not get set.
this is possibly the most finicky part of it all. If anyone on the zendesk sell UI changes these fields (any spelling changes), this will not work.